examples/api/lib/widgets/hardware_keyboard/key_event_manager.0.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+// This example app demonstrates a use case of patching
+// `KeyEventManager.keyMessageHandler`: be notified of key events that are not
+// handled by any focus handlers (such as shortcuts).
+//
+// See [KeyEventManager.keyMessageHandler].
```

Comment:
This won't link in the IDE, so you don't need the []

Also, Because we don't want to maintain two copies of the description that can get out
of date, we tend not to put detailed descriptions here, other that to say what it is
attached to. So, I'd change this to:

```suggestion
/// Flutter code sample for [KeyEventManager.keyMessageHandler].
```